ratbagd/ratbagd-resolution.c

```diff
@@ -102,14 +103,20 @@ static int ratbagd_resolution_set_active(sd_bus_message *m,
 	int r;
 
 	r = ratbag_resolution_set_active(resolution->lib_resolution);
-	if (r < 0)
-		return r;
+	if (r < 0) {
+		sd_bus *bus = sd_bus_message_get_bus(m);
+		sd_bus_emit_signal(bus,
+				   ratbagd_device_get_path(resolution->device),
+				   RATBAGD_NAME_ROOT ".Device",
+				   "Resync",
+				   NULL);
+	}
```

Comment:
I'm gonna be a pain here but:
1. please squash the "fix return values" patches into one patch. They're the same thing, there's no point separating them by file barrier
2. add a `ratbagd_device_resync(device, m)` helper function and call that so you don't need the same boilerplate everywhere. This make it easier to do extra resync work in that function in the future

Reply:
> I'm gonna be a pain here

That's what code reviews are for :) Take another look, it should be good now!